tee/prototype/epochproc.go

```diff
@@ -122,7 +122,6 @@ func (e *Enclave) txRoutine(
 			err := e.applyEpochTx(txEpoch, tx)
 			if err != nil {
 				log.Errorf("txRoutine: Error applying tx: %v", err)
```

Comment:
do `log := log.WithFields` in line 115 then we have nice context information here ;)